runtime/common/src/main/java/org/apache/nemo/runtime/common/metric/MetricUtils.java

```diff
@@ -294,23 +291,4 @@ private static String recursivelyFindLicense(final Path path) {
       throw new MetricException(e);
     }
   }
-
-  /**
-   * De-register Beam JDBC driver, which produces inconsistent results.
-   */
-  public static void deregisterBeamDriver() {
-    final String beamDriver = "org.apache.beam.sdk.extensions.sql.impl.JdbcDriver";
-    final Enumeration<Driver> drivers = DriverManager.getDrivers();
-    while (drivers.hasMoreElements()) {
-      final Driver d = drivers.nextElement();
-      if (d.getClass().getName().equals(beamDriver)) {
-        try {
-          DriverManager.deregisterDriver(d);
-        } catch (SQLException e) {
-          throw new MetricException(e);
-        }
-        break;
-      }
-    }
-  }
```

Comment:
What's this?

Reply:
This was a patch for the bug on the Beam side, but it's been resolved after the 2.10.0 update. Here's the link: https://beam.apache.org/blog/2019/02/15/beam-2.10.0.html
Please refer to the line that says: SQL: JDBC driver no longer breaks interactions with other JDBC sources